documentation/endpoints/reactions.md

```diff
@@ -31,3 +32,40 @@ Example Response
   },
 }
 ```
+## Create or update a Reaction v3
+
+```
+POST /api/v3/post/:post_id/reactions
+```
+
+Example Request
+```
+curl http://rogue.dev:8000/api/v3/posts/:post_id/reactions
+ -d '{
+  "northstar_id":5589c991a59dbfa93d8b45ae,
+  }'
+  --header "Accept: application/json"
+  --X-DS-Rogue-API-Key "API_KEY"
+```
+
+Example Response
+```
+{
+    "data": {
+        "northstar_id": "5589c991a59dbfa93d8b45ae",
+        "post_id": "61",
+        "term": {
```

Comment:
Think you can remove this from the docs too.

Reply:
ah yes - thanks @sbsmith86 !